canvas_manage_course/settings/isites_migration.py

```diff
@@ -1,17 +1,32 @@
+"""
+These settings are used for:
+- isites_migration.management.commands.export_slide_tool (just exporting slide
+    tool data; uses e.g.
+- jobs put on the RQ isites_export queue (which run
+    isites_migration.jobs.migrate_files)
+"""
 from .aws import *
 
 INSTALLED_APPS = (
-    'icommons_common',
     'async',
     'django_rq',
+    'icommons_common',
     'isites_migration',
+    'lti_school_permissions',
```

Comment:
Don't mean to dredge up an already approved PR, but I was looking through the code to see if there were any new/modified secure settings files and I have a question about these installed apps.  When using this settings file, why do we now need `manage_people`, `manage_sections`, and `lti_school_permissions`?  Is this related to unit tests?